src/Types/JsonWsp.purs

```diff
@@ -141,7 +158,7 @@ type UtxoQueryResult = Map.Map TxOutRef OgmiosTxOut
 -- TxOutRef
 type TxOutRef =
   { txId :: String
-  , index :: BigInt.BigInt
+  , index :: UInt.UInt
```

Comment:
TxOutRef index is long integer in https://playground.plutus.iohkdev.io/doc/haddock/plutus-ledger/html/Ledger.html#t:TxOutRef

Reply:
In serialization-lib it's just js `Number` though.